ofrak_core/test_ofrak/components/test_filesystem_component.py

```diff
@@ -397,3 +407,25 @@ def diff_directories(dir_1, dir_2, extra_diff_flags):
             second_type = " ".join(second.split(" ")[4:])
 
             assert first_type == second_type
+
+
+class TestCPIOFilesystem:
+    async def test_unpack(self, cpio_filesystem_root):
```

Comment:
Can we have a docstring that descriptes what this tests? (/dev/console) seems to be it.

Reply:
Also, can we move this test to `test_cpio_component.py` to match all other compressed filesystem tests?